foundation-ui/routing/RoutingService.js

```diff
@@ -1,39 +1,48 @@
 import {Route, Redirect} from 'react-router';
 import {EventEmitter} from 'events';
 
-import EventTypes from './EventTypes';
+import {ROUTING_CHANGE} from './EventTypes';
+
+function throwError(error) {
+  if (global.__DEV__) {
```

Comment:
Let's agree on a consistent way identifying dev environments. Either we use the React way (`process.env.NODE_ENV`)  or we introduce a global like the one you're using. /cc @dcos/frontend